eth/tracers/tracers_test.go

```diff
@@ -298,3 +344,103 @@ func jsonEqual(x, y interface{}) bool {
 	}
 	return reflect.DeepEqual(xTrace, yTrace)
 }
+
+// Iterates over all the input-output datasets in the tracer parity test harness and
+// runs the JavaScript tracers against them.
+func TestCallTracerParity(t *testing.T) {
+	files, err := ioutil.ReadDir("testdata")
+	if err != nil {
+		t.Fatalf("failed to retrieve tracer test suite: %v", err)
+	}
+	for _, file := range files {
+		if !strings.HasPrefix(file.Name(), "parity_call_tracer_") {
```

Comment:
Instead of `ioutil.Readdir` followed by this filter in the next stanza, it might be more legible to use `filepath.Glob("testdata/parity_call_tracer*")`.

Reply:
Wonderful idea. At this point I tried to do the minimal changes from existing code in order to later make a PR at foundation/upstream. This test is an almost exact copy of https://github.com/etclabscore/core-geth/pull/255/files/b194b2c4f83ae3f15927accec95e01bc391e68fe#diff-6b1dc15800ea95ec572b25668c1a2f58ebae907d561623718f54e6a6cb7977e7R251

Do you think keeping a copy without additional changes helps on PRs to upstream?

Reply:
Gotch, yeap, that sounds fine to me. Maybe we can do a tiny `nice-to-have` PR for the globber later ;)